addon/popup.jsx

```diff
@@ -279,15 +279,15 @@ class MailPreference extends React.Component {
 
       <div className="providers-container">
         <button className="provider-button" onClick={this.onSelect.bind(this, "gmail")}>
-          <img className="provider-icon" src="images/gmail.svg"/>
+          <img className="provider-icon" src="images/gmail.svg" alt="gmail icon"/>
           <p>Gmail</p>
         </button>
         <button className="provider-button" disabled={!showAdditionalProviders} onClick={providerCb.bind(this, "yahoo")}>
-          <img className="provider-icon" src="images/yahoo.svg"/>
+          <img className="provider-icon" src="images/yahoo.svg" alt="yahoo icon"/>
           <p>Yahoo Mail</p>
         </button>
         <button className="provider-button" disabled={!showAdditionalProviders} onClick={providerCb.bind(this, "outlook")}>
-          <img className="provider-icon" src="images/outlook.svg"/>
+          <img className="provider-icon" src="images/outlook.svg" alt="outlook icon"/>
```

Comment:
These should be `alt=""` – it's the text replacement, and in this case we don't need a text replacement for the icon since the text is below. If the attribute is left out then it's unclear what's intended, but if it's empty it should be good.

Reply:
:+1: